go-sqlsmith/dml_sql.go

```diff
@@ -12,25 +12,26 @@ import (
 // SelectStmt make random select statement SQL
 func (s *SQLSmith) SelectStmt(depth int) (string, error) {
 	tree := s.selectStmt(depth)
-	return s.Walk(tree)
+	stmt, _, err := s.Walk(tree)
+	return stmt, err
 }
 
 // UpdateStmt make random update statement SQL
-func (s *SQLSmith) UpdateStmt() (string, error) {
+func (s *SQLSmith) UpdateStmt() (string, string, error) {
 	tree := s.updateStmt()
 	return s.Walk(tree)
 }
 
-// InsertStmtAST implement insert statement from AST
-func (s *SQLSmith) InsertStmtAST() (string, error) {
+// InsertStmt implement insert statement from AST
+func (s *SQLSmith) InsertStmt(fn bool) (string, string, error) {
 	tree := s.insertStmt()
 	return s.Walk(tree)
 }
 
-// InsertStmt make random insert statement SQL
-func (s *SQLSmith) InsertStmt(fn bool) (string, error) {
+// InsertStmtStr make random insert statement SQL
+func (s *SQLSmith) InsertStmtStr(fn bool) (string, string, error) {
```

Comment:
Seems there is no function will call InsertStmtStr?

Reply:
Yes, but there are 2 ways of generate insert statement, from AST and building from string directly. Just keep both of them here.